tests/test_settings.py

```diff
@@ -117,3 +117,24 @@ def test_get_callback_function_with_non_callable_string(self, import_string_mock
     def test_get_callback_function_(self):
         with self.assertRaises(ImportError):
             get_callback_function("DJSTRIPE_TEST_CALLBACK")
+
+
+class TestGetApiVersion(TestCase):
+
+    @override_settings(DJSTRIPE_API_VERSION=None)
+    def test_with_default(self):
+        self.assertEquals('2017-02-14', get_api_version())
```

Comment:
True, but it also tests that the default value is what we *think* it is. ;-)  I think it should remain as a sanity check - updating the API version shouldn't be that easy!